screens/Registration/InformationScreen/InformationScreen.tsx

```diff
@@ -35,43 +34,51 @@ const InformationScreen: React.FC = () => {
   const useNav = useNavigation()
   const keyboardVisible = useKeyboardVisibility()
 
-  const { firstName, setFirstName, lastName, setLastName, date, setDate, location, setLocation, description } = useContext(RegistrationContext)
+  const {
+    firstName,
+    setFirstName,
+    lastName,
+    setLastName,
+    date,
+    setDate,
+    location,
+    setLocation,
+    description,
+  } = useContext(RegistrationContext)
 
   const onPress = () => {
     setDateModal(true)
     setHasBeenTouched(true)
   }
 
   const checkFields = () => {
-    if(firstName === ""){
+    if (firstName === "") {
       showErrorToast("You need to input your first name!")
       return false
     }
-    if(lastName === ""){
+    if (lastName === "") {
       showErrorToast("You need to input your last name!")
       return false
     }
-    if(!hasBeenTouched){
+    if (!hasBeenTouched) {
       showErrorToast("You need to input your birth day!")
       return false
-    } 
+    }
     return true
   }
 
   const opacity = !hasBeenTouched ? 0.2 : 1
 
   return (
-    <TouchableWithoutFeedback 
-      style={[
-        styles.container, 
-        {height: Dimensions.get('window').height + insets.bottom + insets.top}
-      ]}
-      onPress={() => Keyboard.dismiss()}>
```

Comment:
make sure that deleting stylings depending on the window's dimensions is not breaking the layout.